CHANGELOG.md

```diff
@@ -12,6 +12,9 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
   [cyberark/conjur-service-broker#306](https://github.com/cyberark/conjur-service-broker/pull/306)
 
 ### Security
+- Update rack in Gemfile.lock and tests/integration/test-app/Gemfile.lock to 2.2.6.4 to address CVE-2023-27539,
+  and activesupport in Gemfile.lock and tests/integration/test-app/Gemfile.lock to 6.1.7.3
```

Comment:
```suggestion
  and activesupport in Gemfile.lock and tests/integration/test-app/Gemfile.lock to 6.1.7.3 for CVE-2023-28120 (not vulnerable)
```

In this case, I evaluated the CVE and we're pretty likely not vulnerable to the issue in activesupport (we don't use the new bytesplice method in Ruby 3.2) so it's OK to add the (not vulnerable) for the second CVE.